test/integration/functional_test.go

```diff
@@ -239,6 +240,25 @@ func validateStartWithProxy(ctx context.Context, t *testing.T, profile string) {
 	if !strings.Contains(rr.Stderr.String(), want) {
 		t.Errorf("start stderr=%s, want: *%s*", rr.Stderr.String(), want)
 	}
+
+	l, err := auditLastLine()
```

Comment:
here try to add subtests, so when this fails, in gopogh it will show as
/TestFunction/Audit....

Reply:
Added subtest